### ISSUE TITLE
Intracranial hypertension

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#7479]
**Preferred term label:**
Intracranial hypertension

**Synonyms**
Pseudotumor cerebri

**Definition (free text, please give PubMed ID)**
Raised intracranial pressure is found in a number of disorders caused by increased production, decreased absorption, or altered CSF dynamics. Historically, this was most often a sign of a space occupying lesion in the confines of the cranium. Now many cases are a sign of idiopathic intracranial hypertension, a cause of headaches and visual symptoms and impairment commonly associated with excess obesity or as an adverse affect of drugs.

PMID: 29903905
**Parent term (use hpo.jax.org/app)**

Abnormality of the cerebrospinal fluid HP:0002921

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Idiopathic intracranial hypertension, Pseudotumor cerebri, Benign intracranial hypertension, Brain tumor, Brain abscess, Meningitis

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
This term already exists, [Increased intracranial pressure HP:0002516](https://hpo.jax.org/app/browse/term/HP:0002516).

I will add Pseudotumor cerebri as a RELATED synonym, because it is restricted to idiopathic etiology, which should not be part of the HPO term.
